src/repositories/jobRepository.ts

```diff
@@ -122,8 +122,8 @@ export class JobRepository extends BaseRepository {
       options,
       `Mongo Timeout Error: Timed out while retrieving job`
     );
-    if (response.value) {
-      const job: Job = response.value;
+    if (response) {
+      const job: Job = response;
```

Comment:
We should change this back!

Reply:
@mmeigs even when all the tests are working? I guess you are right, this will keep things consistent.